extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -125,7 +125,9 @@ protected function requestParsoid( $method, $title, $params ) {
 			}
 			\Wikia\Logger\WikiaLogger::instance()->error( 'ApiVisualEditor_requestParsoid', [
 				'method' => $method,
-				'error' => 'parsoidserver-' . $code
+				'error' => 'parsoidserver-' . $code,
```

Comment:
Would it make sense to rename it to error_code and remove the 'parsoidserver-' part?